### PR TITLE
Use standard 21 port for new FTP deployment

### PR DESCRIPTION
**Pull request by sbesson**
- define a single FTP security group
- add the 21 port to the FTP security group

### CODE REVIEW

File: ansible/openstack-create-ftp.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@
   - name: IDR FTP external access security group
     os_security_group:
       description: External access to IDR FTP (managed by Ansible)
-      name: "{{ idr_environment_idr }}-ftp-external"
```
Comment (joshmoore) [inline ansible/openstack-create-ftp.yml:19]: Can you explain this change?

**Review by joshmoore — APPROVED** [review]
One question, but generally looks fine.

File: ansible/openstack-create-ftp.yml (hunk 1 above)
Comment (sbesson) [reply ansible/openstack-create-ftp.yml:19]: I think the goal of the existing implementation was to be able to have multiple FTP servers e.g. production and staging for testing and test security group changes without affecting production.

This change is effectively mirroring the naming of the other IDR security groups defined in https://github.com/IDR/ansible-role-openstack-idr-security-groups/blob/master/tasks/main.yml . This assumes the settings should be considered as stable API. A potential next step if we are happy with this approach would be to migrate the security group creation upstream to the Ansible role.

Comment (joshmoore) [reply ansible/openstack-create-ftp.yml:19]: :+1:

**Pull request merged**